lib/ui/screens/contact_detail_screen.dart

```diff
@@ -1,20 +1,120 @@
+                                     fontWeight: FontWeight.w500, fontSize: 35))
+        ]; 
+    } 
+
+    List<String> contactInfoList(Contact contactInfo){
+        String gender = contactInfo.gender == 1? "Male" : 
+                        (contactInfo.gender == 2 ? "Female" : ""); 
+        return <String> [contactInfo.firstName, contactInfo.lastName,
+                        contactInfo.address, gender, contactInfo.email];  
+        
+    }
+
+    Text titleInfo(String title){
+        return Text(
+            title, 
+            style: TextStyle(fontFamily: 'LobsterTwo', fontSize: 22)
+        );  
+    }
+
+    List<Widget> titleObject(String title){
+        return [
+            SizedBox(height: 20), 
+            titleInfo(title), 
+            divider, 
+        ]; 
+    }
+
+    Row detailInfo(String title, String info){
+        return Row(
+            mainAxisAlignment: MainAxisAlignment.spaceAround,
+            children: [
+                Spacer(),
+                Expanded(
+                    child: Row(
+                        mainAxisAlignment: MainAxisAlignment.start,
+                        children: [
+                            Text(title + ": ", 
+                            style: TextStyle(fontFamily: 'Peddana', fontSize: 25, fontWeight: FontWeight.w700))
+                        ]
+                    )
+                ),
+                Expanded(
+                    child: Row(
+                        mainAxisAlignment: MainAxisAlignment.end,
+                        children: [
+                            Text(info, 
+                            style: TextStyle(fontFamily: 'Peddana', fontSize: 25))
+                    ]
+                    )
+                ),
+                Spacer()
+            ]
+        ); 
+    }
+
+    List<Row> detailList(List<String> infoList){
+        List<Row> detailList = []; 
+        for (int i = 0; i<infoList.length; i++)
+            detailList.add(detailInfo(fields[i], infoList[i])); 
+        return detailList; 
+    }
+
+    Widget bodyContactDetail(List<String> infoList, String phone, BuildContext context){
+        List<Widget> colChildren = []; 
+        colChildren.addAll(titleObject("Phone")); 
+        colChildren.add(detailInfo("Phone Number", phone)); 
+        colChildren.addAll(titleObject("Detail")); 
+        colChildren.addAll(detailList(infoList)); 
+        return Column(
+            children: colChildren,
+        ); 
+    }
+
     @override
     Widget build(BuildContext context) {
-        var contactName = contact.firstName + " " + contact.lastName + contact.id.toString();
+        //get full information of contact by id 
+        Contact contactInfo = MockContact.mockContact(contact.id); 
+        String contactName = contactInfo.firstName + " " + contactInfo.lastName + contactInfo.id.toString();
+        List<String> infoList = contactInfoList(contactInfo); 
+        String phone = contactInfo.phoneNumber; 
         return Scaffold(
```

Comment:
Try the Scaffold with attribute `extendBodyBehindAppBar: true,` to get the full screen display effect.